store/types/errors.go

```diff
@@ -1,9 +1,42 @@
 package types
 
 import (
-	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	errorsmod "cosmossdk.io/errors"
+	abci "github.com/tendermint/tendermint/abci/types"
 )
 
 const StoreCodespace = "store"
 
-var ErrInvalidProof = sdkerrors.Register(StoreCodespace, 2, "invalid proof")
+var ErrInvalidProof = errorsmod.Register(StoreCodespace, 2, "invalid proof")
+
+var (
```

Comment:
errors moved here are meant to separate root sdk errors from storage errors. 

Should we merge snapshot errors into store errors or leave it separate?